tests/cpp/zeek.cc

```diff
@@ -10,13 +10,74 @@
 #include <utility>
 
 #include "broker/data.hh"
+#include "broker/time.hh"
 
 using namespace broker;
+using namespace std::chrono_literals;
 
 TEST(event) {
   auto args = vector{1, "s", port(42, port::protocol::tcp)};
   zeek::Event ev("test", vector(args));
   zeek::Event ev2(std::move(ev));
+  REQUIRE(ev2.valid());
   CHECK_EQUAL(ev2.name(), "test");
+  CHECK_EQUAL(ev2.ts(), std::nullopt);
   CHECK_EQUAL(ev2.args(), args);
 }
+
+TEST(event_ts) {
+  auto args = vector{1, "s", port(42, port::protocol::tcp)};
+  zeek::Event ev("test", vector(args), broker::timestamp(12s));
+  zeek::Event ev2(std::move(ev));
```

Comment:
This test claims to be about the event timestamp. Why is it moving for no apparent reason? Just copy & paste from the `event` test? That test should also be split, looking at it.

Reply:
Yeah, copy/paste. I've tried to reorder things and removed the unnecessary and repeated `args` as well.